packages/graphql/src/api-v6/queryIRFactory/ReadOperationFactory.ts

```diff
@@ -143,8 +146,10 @@ export class ReadOperationFactory {
         const sortInputFields = this.getSortInputFields({ entity: relTarget, relationship, sortArgument });
 
         const firstArgument = connectionTree.args.first;
+        const afterArgument = connectionTree.args.after ? cursorToOffset(connectionTree.args.after) : undefined;
```

Comment:
As this will be duplicated between top-level and nested, we could handle it in a separate method `getPagination`